### PR TITLE
Added toggle button 

### PR DESCRIPTION
**Pull request by Jdure**

### CODE REVIEW

**Comment by Jdure** [comment]
Commented out the NPM run audio was running into issues w/pipeline.

File: .github/workflows/main.js.yml [hunk 1]
```diff
@@ -29,10 +29,10 @@ jobs:
...
         echo "::endgroup::"
     - run: npm ci
-    - run: npm run audio
-      env:
-        APIKEY: ${{ secrets.SPEECH_KEY }}
-        REGION: ${{ secrets.SPEECH_REGION }}
+    # - run: npm run audio
+    #   env:
+    #     APIKEY: ${{ secrets.SPEECH_KEY }}
+    #     REGION: ${{ secrets.SPEECH_REGION }}
```
Comment (e-wu) [inline .github/workflows/main.js.yml]: Any reason why this commented out?
Pinging @gdifrusc

Comment (Jdure) [reply .github/workflows/main.js.yml]: Commented out the NPM run audio, I was running into issues w/pipeline. @gdifrusc

Comment (Jdure) [reply .github/workflows/main.js.yml]: The code has been re-added

**Pull request merged**